test/test_dcp.cpp

```diff
@@ -101,10 +101,10 @@ TEST_F (DcpTest, DcpHelloTest)
 
    p_buf = pnal_buf_alloc (PF_FRAME_BUFFER_SIZE);
    memcpy (p_buf->payload, get_name_req, sizeof (get_name_req));
-   ret = pf_eth_recv (net, p_buf);
+   ret = mock_pnal_eth_recv (net, p_buf);
```

Comment:
What is being tested here? Looks like it is a test of the mock-function.

Reply:
I have updated the the mock implementation to avoid this.
Done